src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -3469,7 +3484,7 @@ def test_fs_anyspace_and_readwrite_error():
                 in str(excinfo.value))
 
 
-def test_halo_exchange_view(capsys):
+def test_halo_exchange_view(tmpdir,capsys):
```

Comment:
pycodestyle error. In fact, it looks as though `tmpdir` isn't used in this test?

Reply:
Removed.